com.tominlab.wonderpen.metainfo.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.tominlab.wonderpen</id>
+  <launchable type="desktop-id">com.tominlab.wonderpen.desktop</launchable>
+  <name>WonderPen</name>
+  <name xml:lang="zh-cn">妙笔</name>
+  <developer_name>TominLab</developer_name>
+  <summary>Writing novels, essays, etc.</summary>
+  <summary xml:lang="zh-cn">强大易用的写作软件</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://www.tominlab.com/en/terms/service</project_license>
```

Comment:
Does this allow redistribution?

Reply:
@oldj is the author of this application

Reply:
Any way to verify? I don't see any mention of them in tominlab.com

Reply:
Yes, @oldj is the author of this application. My verified email is support@tominlab.com, can this prove it?

Reply:
If you can link, @oldj's Github account as the developer somewhere in https://www.tominlab.com/en/, that would be enough.

Reply:
We have added a new simple developers page: https://www.tominlab.com/en/pages/developers , I am the only developer at the moment.

Reply:
Thanks that'd do